jujugui/static/gui/src/app/components/terminal/terminal.js

```diff
@@ -112,9 +112,12 @@ class Terminal extends React.Component {
             }
           }
       }
-      if (resp.code === 'ok' && resp.message === 'session is ready') {
+      if (resp.code === CODE_OK && (
+        resp.message === 'session is ready' || // Legacy API.
```

Comment:
It's not even 'released' yet and we're already supporting a legacy API? Lets just delete this.

Reply:
Well, we made a blog post describing how to set up your own shell, and we know people that did it, so, given how easy is to be backward compatible in this case, I'd say let's at least for a while. What do you think?